### DISCUSSION TITLE
Error 1105: Exception, msg: java.lang. Exception: Node catalog is not ready, please wait for a while.

### DISCUSSION THREAD

**Discussion by TheThirdReich77** [apache/doris#36396] [A - General / Q&A]
Error 1105: Exception, msg: java.lang. Exception: Node catalog is not ready, please wait for a while.
Doris Version：2.0.9

This error is returned ocassionally by fe when doing some query via jdbc. And then it's resolved automatically. Is there any backend sync or refresh task executing about fe's metadata? How can I avoid this error?

**Comment by TheThirdReich77** [comment]
Is there anyone could help me please?

**Comment by zy-kkk** [comment]
This problem is usually caused by frequent writes to the master FE node, which results in the follower FE replayer being unable to consume within a certain period of time.
If the follower replayer finds that the current time is too different from the synchronizedTimeMs of the previous one, it sets itself to the not ready state; you can consider increasing the meta_delay_toleration_second config of the follower FE.

In addition, if you use the External Catalog function of Doris, please check whether the Catalog has metadata_refresh_interval_sec property. If so, please try to increase this time to reduce the pressure on DorisFE's own metadata writing caused by frequent acquisition of external metadata.
